code/__DEFINES/qdel.dm

```diff
@@ -13,3 +13,6 @@
 #define GC_QUEUED_FOR_QUEUING -1
 #define GC_QUEUED_FOR_HARD_DEL -2
 #define GC_CURRENTLY_BEING_QDELETED -3
+
+#define qdeleted(X) (istype(X, /datum) && X.gc_destroyed)
```

Comment:
I don't like macros pretending to be procs (non-UPPERCASE macro names)
It should be evident when you use a macro that it's a macro, and we do that via UPPERCASE.